hax-lib-macros/src/syn_ext.rs

```diff
@@ -7,15 +7,15 @@ pub struct ExprClosure1 {
     pub body: syn::Expr,
 }
 
-pub trait PatExt {
-    // Make sure to remove type ascriptions
-    fn untype(mut pat: syn::Pat) -> syn::Pat {
-        if let syn::Pat::Type(sub) = pat {
-            pat = *sub.pat.clone();
-        }
-        pat
-    }
-}
+// pub trait PatExt {
+//     // Make sure to remove type ascriptions
+//     fn untype(mut pat: syn::Pat) -> syn::Pat {
+//         if let syn::Pat::Type(sub) = pat {
+//             pat = *sub.pat.clone();
+//         }
+//         pat
+//     }
+// }
```

Comment:
These were unused, idk if you want to keep them around

Reply:
No, let's get rid of that!